src/main/resources/templates/fragments.html

```diff
@@ -11,9 +11,9 @@
 <nav th:fragment="header">
     <h1 th:text="${title}">Ingredients</h1>
     <ul class="nav">
-        <li class="nav-item"><a class="nav-link" href="/events">All Ingredients</a></li>
-        <li class="nav-item"><a class="nav-link" href="/events/create">Add Ingredient</a></li>
-        <li class="nav-item"><a class="nav-link" href="/events/delete">Delete Ingredients</a></li>
+        <li class="nav-item"><a class="nav-link" href="/ingredients">All Ingredients</a></li>
+        <li class="nav-item"><a class="nav-link" href="/ingredients/add">Add Ingredient</a></li>
+        <li class="nav-item"><a class="nav-link" href="/ingredients/delete">Delete Ingredients</a></li>
```

Comment:
This one's kinda obvious, but your links in the header weren't pointing to the right page. 😄